src/EditorFeatures/Core/Implementation/RenameTracking/RenameTrackingTaggerProvider.TrackingSession.cs

```diff
@@ -172,6 +179,42 @@ public void Cancel()
                 return TriggerIdentifierKind.NotRenamable;
             }
 
+            private async Task<TriggerIdentifierKind> DetermineIfRenamableSymbolsAsync(IEnumerable<ISymbol> symbols, Document document, SyntaxToken token)
+            {
+                if (!symbols.Any())
+                {
+                    return TriggerIdentifierKind.NotRenamable;
+                }
```

Comment:
Remove this check. There are always symbols (guaranteed above, and we could assert that this is true).